### PR TITLE
chore: enable perspective bookmarks on lower env EXLM-1937

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1937

Test URLs::

- Before: https://main--exlm--adobe-experience-league.hlx.live/
- After: https://exlm-1937--exlm--adobe-experience-league.hlx.live/

### CODE REVIEW

File: blocks/article-marquee/article-marquee.js [hunk 1]
```diff
@@ -52,7 +58,7 @@ async function createOptions(container, readTimeText) {
     container: options,
     id: window.location.pathname,
     link: window.location.href,
-    bookmarkConfig: false,
+    bookmarkConfig: isPerspectivesBookmarkEnabled(),
```
Comment (ruhisingh1) [inline blocks/article-marquee/article-marquee.js:61]: we can directly use bookmarkConfig: !isProd,

File: scripts/browse-card/browse-card.js [hunk 2]
```diff
@@ -8,6 +8,21 @@ import { CONTENT_TYPES } from '../data-service/coveo/coveo-exl-pipeline-constant
...
+  return !isProd; // Bookmarks are enabled only on lower env.
+};
+
+const BOOKMARK_EXCLUSION_CONTENTYPES = [
+  CONTENT_TYPES.LIVE_EVENT.MAPPING_KEY,
+  CONTENT_TYPES.COMMUNITY.MAPPING_KEY,
+  CONTENT_TYPES.INSTRUCTOR_LED.MAPPING_KEY,
+];
+
+if (!isPerspectivesBookmarkEnabled()) {
```
Comment (noormd08) [inline scripts/browse-card/browse-card.js:22]: Add isProd directly.

File: blocks/article-marquee/article-marquee.js (hunk 1 above)
Comment (nitin-rachabathuni) [reply blocks/article-marquee/article-marquee.js]: done done

File: scripts/browse-card/browse-card.js (hunk 2 above)
Comment (nitin-rachabathuni) [reply scripts/browse-card/browse-card.js]: done done

File: scripts/browse-card/browse-card.js [hunk 3]
```diff
@@ -8,6 +8,18 @@ import { CONTENT_TYPES } from '../data-service/coveo/coveo-exl-pipeline-constant
...
+const BOOKMARK_EXCLUSION_CONTENTYPES = [
+  CONTENT_TYPES.LIVE_EVENT.MAPPING_KEY,
+  CONTENT_TYPES.COMMUNITY.MAPPING_KEY,
+  CONTENT_TYPES.INSTRUCTOR_LED.MAPPING_KEY,
+];
+
+if (!isProd) {
+  BOOKMARK_EXCLUSION_CONTENTYPES.push(CONTENT_TYPES.PERSPECTIVE.MAPPING_KEY);
+}
+
```
Comment (noormd08) [inline scripts/browse-card/browse-card.js:22]: Can you please simplify with Spread?

Comment (nitin-rachabathuni) [reply scripts/browse-card/browse-card.js:22]: Okies

Comment (nitin-rachabathuni) [reply scripts/browse-card/browse-card.js]: done Noor

**Review by noormd08 — APPROVED** [review]

**Pull request merged**